### PR TITLE
Initial datamodel/schema for MiriMrsXArtCorrModel

### PR DESCRIPTION
**Pull request by drlaw1558**
**Description**

This PR implements an initial data model and schema for the MIRI MRS Cross Artifact correction reference file ([CRDS-545](https://jira.stsci.edu/browse/CRDS-545))

Checklist
- [x] Tests
- [ ] Documentation
- [x] Change log
- [x] Milestone
- [x] Label(s)

### CODE REVIEW

File: jwst/datamodels/mrsxartcor.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from .reference import ReferenceFileModel
+
+__all__ = ['MiriMrsXArtCorrModel']
+
+
+class MiriMrsXArtCorrModel(ReferenceFileModel):
```
Comment (stscieisenhamer) [inline jwst/datamodels/mrsxartcor.py:6]: Suggest name change to begin with just `Mir` to be `MirMrsXArtCorr`. This is then consistent with other MIRI-specific models.

**Review by hbushouse — APPROVED** [review]
Looks good to me. Only thing missing is the addition of documentation in the reference files section of RTD, but that can come later once this is actually in use.

**Review by stscieisenhamer — CHANGES_REQUESTED** [review]

File: jwst/datamodels/mrsxartcor.py (hunk 1 above)
Comment (stscieisenhamer) [reply jwst/datamodels/mrsxartcor.py:6]: Thought I do note there is an `MiriIFUCubeParsModel`, but others, such as `MirLrsPhotomModel` have just the three letters.

Comment (hbushouse) [reply jwst/datamodels/mrsxartcor.py:6]: I actually see a mixture of "Mir", "Miri", and "MIRI" in use in various datamodel class names, but I think the trend of the more recent ones is in the direction of "Mir".

File: jwst/datamodels/schemas/miri_mrsxartcorr.schema.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/miri_mrsxartcorr.schema"
+title: MIRI MRS Cross Artifact Corrections
+allOf:
+- $ref: referencefile.schema
```
Comment (stscieisenhamer) [inline jwst/datamodels/schemas/miri_mrsxartcorr.schema.yaml:7]: Will need the EXP_TYPE and P_EXP_TY keywords in the schema. These can be inserted using
```
- $ref: keyword_exptype.schema
- $ref: keyword_pexptype.schema
```

**Review by stscieisenhamer — CHANGES_REQUESTED** [review]

File: jwst/datamodels/schemas/miri_mrsxartcorr.schema.yaml (hunk 2 above)
Comment (drlaw1558) [reply jwst/datamodels/schemas/miri_mrsxartcorr.schema.yaml:7]: Added.

**Review by stscieisenhamer — APPROVED** [review]

**Pull request merged**